src/test/java/com/google/api/codegen/testdata/nodejs_main_library.baseline

```diff
@@ -102,6 +88,8 @@ var ALL_SCOPES = [
  *   Custom function to obtain the credentials.
  * @param {grpc.ClientCredentials} opts.sslCreds
  *   A ClientCredentials for use with an SSL-enabled channel.
+ * @param {Object} opts.grpc
```

Comment:
I meant, when and end-user creates an API object, they need to set this parameter to prevent GAX from loading its own grpc module, right? That seems unexpected and potentially like a usability issue; what if the user forgets to set the parameter? Is there some way that we can auto-detect the "right" module?